content/en/docs/concepts/scheduling-eviction/scheduling-framework.md

```diff
@@ -129,17 +129,28 @@ Plugins wishing to perform "pre-reserve" work should use the
 NormalizeScore extension point.
 {{< /note >}}
 
-### Reserve
-
-This is an informational extension point. Plugins which maintain runtime state
-(aka "stateful plugins") should use this extension point to be notified by the
-scheduler when resources on a node are being reserved for a given Pod. This
-happens before the scheduler actually binds the Pod to the Node, and it exists
-to prevent race conditions while the scheduler waits for the bind to succeed.
-
-This is the last step in a scheduling cycle. Once a Pod is in the reserved
-state, it will either trigger [Unreserve](#unreserve) plugins (on failure) or
-[PostBind](#post-bind) plugins (on success) at the end of the binding cycle.
+### Reserve {#reserve}
+
+The Reserve extension point backs two informational scheduling phases called
+Reserve and Unreserve which are implemented in pairs and get called during these
+phases. Plugins which maintain runtime state (aka "stateful plugins") should use
+these extension points to be notified by the scheduler when resources on a node
+are being reserved and unreserved for a given Pod.
+
+The Reserve phase happens before the scheduler actually binds the Pod to the
+designated node. It exists to prevent race conditions while the scheduler waits
+for the bind to succeed. The Reserve method of each Reserve plugin may succeed
+or fail; if one plugin fails, subsequent plugins are not executed and the
+extension point is considered to have failed. If all plugins' Reserve methods
+succeed, the extension point is considered to be successful and the rest of the
+scheduling cycle and the binding cycle are executed.
+
+The Unreserve method of a Reserve plugin is executed if the Reserve extension
```

Comment:
Something along the lines of `The Unreserve phase is triggered if the Reserve or a later phase fails`

Reply:
Done.